### PR TITLE
1894 upgrade list page with latest designs

### PR DESCRIPTION
**Pull request by solazio**
## Done
- _Upgrade list page with latest designs._

## How to QA
- Visit https://charmhub-io-850.demos.haus/charms and https://charmhub-io-850.demos.haus/bundles
- Compare to [design](https://app.zeplin.io/project/5fb68befd7ea8e4d263f3c26/screen/603f85becb54b6a552c920dd)

## Issue / Card
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1894

## Screenshots
image

image

image

### CODE REVIEW

File: webapp/publisher/views.py [hunk 1]
```diff
@@ -28,12 +28,13 @@ def get_account_details():
     return render_template("publisher/account-details.html")
 
 
-@publisher.route("/charms")
+@publisher.route('/<regex("charms|bundles"):package_type>')
```
Comment (tbille) [inline webapp/publisher/views.py:31]:
```suggestion
@publisher.route('/charms')
@publisher.route('/bundles')
```

No need for regex for this.

File: webapp/publisher/views.py [hunk 2]
```diff
@@ -28,12 +28,13 @@ def get_account_details():
...
 @login_required
-def charms():
+def list_page(package_type):
     publisher_charms = publisher_api.get_account_packages(
         session["publisher-auth"], "charm"
     )
+    print(publisher_charms)
```
Comment (tbille) [inline webapp/publisher/views.py:37]: hello there :smile:

File: webapp/publisher/views.py (hunk 1 above)
Comment (tbille) [reply webapp/publisher/views.py:31]: The filtering can be done directly in the template.

**Comment by spencerbygraves** [comment]
@solazio Please can you add the header as in the [design](https://zpl.io/a88jedX). Do we not have the latest release and version columns?

**Comment by danielmutis** [comment]
@solazio the position of the info icon seems off. It looks off centre from that text.

**Comment by solazio** [comment]
@danielmutis @spencerbygraves updated
image

**Comment by spencerbygraves** [comment]
@solazio can we keep the visibility columns aligned, as you had before? Thanks.

**Comment by solazio** [comment]
@spencerbygraves sure.
image

**Comment by spencerbygraves** [comment]
@solazio great, thanks

File: templates/publisher/list.html [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+  </div>
+</section>
+<section class="p-strip u-extra-space is-shallow">
+  <div class="u-fixed-width">
+    <ul class="p-inline-list--stretch is-navigation u-no-margin--bottom">
+      <li class="p-inline-list__item">
+        <nav class="p-tabs" aria-label="Charms and bundles navigation">
+          <ul class="p-tabs__list" role="tablist">
+            <li class="p-tabs__item" role="presentation">
+              <a href="/charms" class="p-tabs__link" tabindex="0" role="tab" aria-controls="charms" {% if request.path.endswith('charms') %}aria-selected="true" {% endif %}>Charms</a>
```
Comment (tbille) [inline templates/publisher/list.html:19]: You should be able to replace `request.path.endswith('charms')` with `page_type`

Comment (solazio) [reply templates/publisher/list.html]: done

**Review by jkfran — APPROVED** [review]

**Pull request merged**